src/qtoolkit/io/sge.py

```diff
@@ -280,6 +293,10 @@ def _get_qstat_base_command(self) -> list[str]:
     def _get_job_ids_flag(self, job_ids_str: str) -> str:
         return f"-j {job_ids_str}"
 
+    def _get_job_cmd(self, job_id: str):
+        cmd = f"qstat -j -xml {job_id}"
```

Comment:
the last thing that I am not sure if I understand it correctly is if I shall change this to:
```suggestion
    def _get_job_cmd(self, user: str):
        cmd = f"qstat -u -xml {user}"
```

to make it entirely impossible to query by (a list of) job ID(s)

Reply:
Thanks for the updates. Concerning this point, this is already defined in the original base class (`BaseSchedulerIO`) and the input is a job_id. So not to break the inheritance here I would say that it should stay that way. I propose to also raise here, saying that this option is not supported for SGE